templates/repo/actions/runs_list.tmpl

```diff
@@ -36,6 +36,15 @@
 					<div class="run-list-meta">{{svg "octicon-calendar" 16}}{{DateUtils.TimeSince .Updated}}</div>
 					<div class="run-list-meta">{{svg "octicon-stopwatch" 16}}{{.Duration}}</div>
 				</div>
+				{{if and ($.AllowDeleteWorkflowRuns) (.Link) (.Status.IsDone)}}
```

Comment:
Why check `(.Link)`?

Reply:
Hmm, I do not think the old code is right (since Implement actions (#21937))

the `run.Link` should be used directly without check.

Reply:
IMO the old `$.Link` is an abuse (over-defensive programming)
.... I think we should always use `run.Link` in this case (just forget the `$.Link`).